packages/office-ui-fabric-react/src/components/Link/Link.tsx

```diff
@@ -48,7 +48,15 @@ export class Link extends BaseComponent<ILinkProps, any> implements ILink {
             onClick={ this._onClick }
             ref={ this._resolveRef('_link') }
           >
-            { children }
+            <a href=""
```

Comment:
A link without an href should be rendered as a button. You can't render an A tag in a button...